packages/nl-backlink-component/style.css

```diff
@@ -1,23 +1,23 @@
 .nl-backlink {
-	color: var(--nl-backlink-color, var(--nih-link-color, black));
-	text-decoration: none;
+  color: var(--nl-backlink-color, var(--nih-link-color, hsl(0 0% 0%)));
```

Comment:
I would propose not to restrict people to the use of hsl colors only, as it's not a super common web standard like rgb or hex. I agree hsl might have some benefits, but let's suggest it rather then prevent other options :)